### ISSUE TITLE
Show number of search results

### ISSUE THREAD

**Issue by steffenkleinle** [digitalfabrik/integreat-app#2520]
**Is your feature request related to a problem? Please describe.**
As a user using the search I want to know how many results are found.

**Describe the solution you'd like**
This is relevant on both web and native and probably for both the normal category search and the city search. Somewhere a number of search results should be shown. Or do you disagree and think the number of search results is not necessary?

<img src="https://github.com/digitalfabrik/integreat-app/assets/29095487/206c5384-ca62-4468-bd26-956edde6c1a0" width="200px" />
<img src="https://github.com/digitalfabrik/integreat-app/assets/29095487/4e2da6ec-686f-46b4-8be4-0a789d0438e7" width="200px" />

**Describe alternatives you've considered**
None.

**Additional context**
Relates to #2352.

**Comment by nikolahoff** [comment]
@steffenkleinle can you tell me what these items are that are shown when no search term has been entered yet?
Is it possible in this case to shoe "most searched" items?
![Uploading Bildschirmfoto 2024-05-14 um 12.18.54.png…]()

**Comment by nikolahoff** [comment]
<img width="1279" alt="Bildschirmfoto 2024-05-14 um 12 18 54" src="https://github.com/digitalfabrik/integreat-app/assets/97434837/ee62afff-6bc3-4ca8-85da-30b3393df150">

**Comment by steffenkleinle** [comment]
> Ok, I think then we don't need to implement it. What was the reason for implementing the number of search results? @steffenkleinle

I think it was somehow related to accessibility/have a better overview. However, if you think it does not make sense from an UI/UX perspective we can also just reject this :)

**Comment by nikolahoff** [comment]
@steffenkleinle [here](https://www.figma.com/design/cA4F2MwHs2LNGviOWUjfE9/Integreat-Tickets-Frontend?node-id=2944-794&t=brGhCmrpcn6PQWxj-1) is the link to the design for web and native.

**Comment by steffenkleinle** [comment]
> @steffenkleinle [here](https://www.figma.com/design/cA4F2MwHs2LNGviOWUjfE9/Integreat-Tickets-Frontend?node-id=2944-794&t=brGhCmrpcn6PQWxj-1) is the link to the design for web and native.

Looks great, thanks. No comments or requests from my side :)
